uk.co.compucorp.civicrm.hrcore/CRM/HRCore/APIWrapper/DefaultLimitRemover.php

```diff
@@ -5,7 +5,11 @@ class CRM_HRCore_APIWrapper_DefaultLimitRemover implements API_Wrapper {
   /**
    * In civicrm API, 0 means there is no limit on the retrieved results
    */
-  const NO_LIMIT_ON_RESULTS = 0;
+  static private $DEFAULT_NO_LIMIT_VALUE = 0;
```

Comment:
Since this is the default value for the Limit option, I think it makes more sense to name the variable as `DEFAULT_API_LIMIT_VALUE`.

Also, what's the reason for changing this from a constant to a variable? Did you change it only to make it private?